dockers/docker-config-engine-buster/Dockerfile.j2

```diff
@@ -7,21 +7,16 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update         && \
     apt-get install -y        \
         # Dependencies for sonic-cfggen
-        python-lxml           \
-        python-yaml           \
-        python-bitarray       \
         python-pip            \
+        python3-pip           \
         python-dev            \
+        python3-dev           \
         apt-utils             \
-        python-setuptools
+        python-setuptools     \
+        python3-setuptools
 
 RUN pip install --upgrade pip
-
-RUN pip install             \
-        netaddr             \
-        ipaddr              \
-        jinja2              \
-        pyangbind==0.6.0
+RUN pip3 install --upgrade pip
```

Comment:
@tahmed-dev: You didn't modify the docker-config-engine-buster container makefile to install the Python 3 versions of the packages. I just made a local change to install them, and I did not need to add the pyangbind/enum34 workaround to do so. Hoewever, I am concerned we will see runtime issues. Do you recommend that I add the workaround in the docker-config-engine-buster Dockerfile?

Reply:
Yes, the redis cache would fail as it is using `re` module. I've seen this failure when I was building the Python3 package and so worked around the issue by installing outside the dependencies section of the setuptools.